packages/neptune-extension/src/common/hooks/upgrade.ts

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { useSelector } from 'react-redux'
+
+import { checkVersion } from 'common/api/version';
+
+import { getConfigurationState } from 'common//state/configuration/selectors';
+
+declare var PLATFORM: string
```

Comment:
This is not needed anymore.
I merged my own PR that adds this variable.

Reply:
Ok, thanks, that was the prequisitive.